lib/xboxrt/libc_extensions/direct.h

```diff
@@ -3,3 +3,11 @@
 // SPDX-FileCopyrightText: 2020 Jannik Vogel
 
 // Part of Microsoft CRT
+
+int _chdir(char* path);
+int _chdrive(int drive);
+char* _getcwd(char* buffer, size_t length);
```

Comment:
nit: We don't have a .clang-format file applying to these files yet, but maybe give this file a quick re-format to match it to direct.c which seems to already follow the nxdk convention